internal/streaming/clickhouse_connector/query.go

```diff
@@ -221,21 +242,36 @@ func (d queryMeterView) toSQL() (string, []interface{}, error) {
 	queryView.Select(selectColumns...)
 	queryView.From(viewName)
 
-	if d.Subject != nil {
-		where = append(where, queryView.Equal("subject", *d.Subject))
+	if len(d.Subject) > 0 {
+		mapFunc := func(subject string) string {
+			return queryView.Equal("subject", subject)
+		}
+
+		where = append(where, queryView.Or(slicesx.Map(d.Subject, mapFunc)...))
 	}
+
 	if d.From != nil {
 		where = append(where, queryView.GreaterEqualThan("windowstart", d.From.Unix()))
 	}
+
+	windowEndColumn := "windowstart"
```

Comment:
It's not clear to me why we need this. Can you please comment code.

Reply:
Discussed it offline, this was a misunderstanding on my part. We can use windowend here.